pylint/checkers/format.py

```diff
@@ -602,8 +602,10 @@ def visit_default(self, node: nodes.NodeNG) -> None:
             isinstance(node.parent, nodes.TryFinally) and node in node.parent.finalbody
         ):
             prev_line = node.parent.body[0].tolineno + 1
+        elif isinstance(node.parent, nodes.Module):
+            prev_line = 0
```

Comment:
This is the same effect as doing `node.parent.fromlineno` but avoids the `StatementMissing` exception from calling `statement`.